src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/CreditCardAttribute.cs

```diff
@@ -23,27 +23,26 @@ public override bool IsValid(object value)
                 return true;
             }
 
-            var ccValue = value as string;
-            if (ccValue == null)
+            if (!(value is string ccValue))
```

Comment:
IMO the syntax change is not increasing tremendously readability in this case (the ! throws it off a bit).
I would personally back out this pattern - but @ajcvickers should decide (as well on other things).

Reply:
I like this one. :-)